api/provider/aws/defaults.go

```diff
@@ -121,3 +123,19 @@ const DEFAULT_LB_ROLE_POLICY_TEMPLATE = `{
         }
     ]
 }`
+
+var DEFAULT_CREATE_LB_PORT_MODEL = []models.Port{
+	models.Port{
+		ContainerPort: 80,
+		HostPort:      80,
+		Protocol:      "tcp",
+	},
+}
+
+var DEFAULT_CREATE_LB_HEALTHCHECK_MODEL = models.HealthCheck{
```

Comment:
You didn't update the `cli/command` package to reference these values. 
While you're doing that, it would be great if you could knock out all default values. 

* Environment OS == "linux"
* Environment Size = "m3.medium"
* Service Scale = 1 (this is new), see my `terraform-update` branch

Reply:
As with the previous comment, I wonder if it would be worth it to open a new issue and open a new PR to address the cli/command that would be required here.

Reply:
The cli portion sure, but you'll need those to exist to finish these unit tests